pyproject.toml

```diff
@@ -20,6 +20,7 @@ dynamic = ["version"]
 
 dependencies = [
     "mqt.bench>=1.1.3",
+    "qiskit!=1.3.2",
```

Comment:
Please add a comment to this exclusion specifying, why this version was excluded.
Otherwise we surely forget that within a couple of days.

Reply:
Done.